project_issue_task_sync/models/project_project.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class project_project(models.Model):
+    _inherit = 'project.project'
+
+
+    sync_tasks_issues = fields.Boolean(
+       string='Sync Issues and tasks',
+       default='feasable_sync'
```

Comment:
you assign a string as default value for a boolean. Works, but only because any nonempty string is truthy. Why not `True`?